tools/ua.bash

```diff
@@ -0,0 +1,33 @@
+# bash completion for ubuntu-advantage-tools
+
+_ua_complete()
+{
+    local cur_word prev_word
+    cur_word="${COMP_WORDS[COMP_CWORD]}"
+    prev_word="${COMP_WORDS[COMP_CWORD-1]}"
+
+    subcmds="attach detach disable enable status version"
```

Comment:
We could dynamically source the optional flags and subcommands with the following: 

subcmds=$(ua --help | awk '/^\s*$|Available|Use/ {next;} /Flags:/{flag=1;next}/Use ubuntu-avantage/{flag=0}flag{ if ( $1 ~ /,/ ) { print $2}  else print $1}')


This only tab-completes the first element, but it's a lot better than nothing.